app/setup-wizard/client/setupWizard.js

```diff
@@ -193,7 +193,19 @@ Template.setupWizard.events({
 				persistSettings(t.state.all(), () => {
 					localStorage.removeItem('wizard');
 					localStorage.setItem('wizardFinal', true);
-					FlowRouter.go('setup-wizard-final');
+
+					if (t.state.get('registerServer')) {
+						Meteor.call('cloud:registerWorkspace', (error) => {
```

Comment:
@sampaiodiego @rodrigok is there a better place I could put this that would be caught even when wizard skipped?

Reply:
watching by `Register_Server` setting value changes on back end is definitely better =)

Reply:
This would be an oplog observe right?  Would this be possible to only have the first time it changes and never again hook the observe up?  I can’t think of how to implement that.

Reply:
> This would be an oplog observe right?

kind of.

> Would this be possible to only have the first time it changes and never again hook the observe up?

do you want to run it only once like even between restarts? like run only once ever

Reply:
It can run per restart... but it'd be wasteful.  I just need to ensure it runs _**at least**_ once that way the process is kicked off so in the future they won't miss push notifications and such.

Reply:
I can move this after we merge.  i'll DM